### PR TITLE
Hypothesis change nav header to contribution #1003

### PR DESCRIPTION
**Pull request by Tosin-Balogun**
## Description

I have made some alternative style changes to our 'community' section based on the hypothesis from our community contribution work. If we changed the 'community' section to either 'contribution' or 'community and contribution', it should improve the information scent for users
### Related issue

Hypothesis: Change nav header to 'contribution' #1003
## Checklist

- [x] Tested against the [NHS.UK frontend testing policy](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/testing.md) (Resolution, Browser & Accessibility)
- [x] Code follows the [NHS.UK frontend coding standards](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/coding-standards.md)
- [x] Version number has been increased in `package.json` (using [SEMVER](https://semver.org/))
- [x] CHANGELOG entry
- [x] Add URL redirects

### CODE REVIEW

**Comment by Tosin-Balogun** [comment]
@davidhunter08 - pushed the latest version after completing url changes.

The links for links do not work though 😞 
- /community-and-contribution/backlog-of-components-and-patterns
- /community-and-contribution/propose-component-pattern
- /community-and-contribution/develop-component-pattern
- /community-and-contribution/contribution-criteria

**Comment by davidhunter08** [comment]
@Tosin-Balogun the link to breadcrumb macro needs updating on the community and contribution pages.

It's currently:

```
{% block breadcrumb %}
  {% include "community/_breadcrumb.njk" %}
{% endblock %}
```

Needs to change to:

```
{% block breadcrumb %}
  {% include "community-and-contribution/_breadcrumb.njk" %}
{% endblock %}
```

**Comment by davidhunter08** [comment]
Looking good @Tosin-Balogun 

Now we need to add some redirects so the old urls (/community) redirect to the new urls (/community-and-contribution).

Each page in `community-and-contribution` needs a redirect.

Redirects go in `app.js` after `line 126`.

For example:

```
app.get('/community', (req, res) => {
  res.redirect('/community-and-contribution');
});

app.get('/community/backlog-of-components-and-patterns', (req, res) => {
  res.redirect('/community-and-contribution/backlog-of-components-and-patterns');
});
```

etc.

File: app.js [hunk 1]
```diff
@@ -133,10 +133,34 @@ app.get('/slack', (req, res) => {
...
+app.get('/community/index', (req, res) => {
+  res.redirect('/community-and-contribution/index');
+});
+
+app.get('/community/propose-component-pattern', (req, res) => {
+  res.redirect('/community-and-contribution/propose-component-pattern');
+});
+
+// REDIRECT STOPS HERE
+
```
Comment (Tosin-Balogun) [inline app.js:163]: Added the redirects for the 5 pages within the community section @davidhunter08

File: app.js [hunk 2]
```diff
@@ -133,10 +133,34 @@ app.get('/slack', (req, res) => {
   res.redirect('https://join.slack.com/t/nhs-service-manual/shared_invite/enQtNTIyOTEyNjU3NDkyLTk4NDQ3YzkwYzk1Njk5YjAxYTI5YTVkZmUxMGQ0ZjA3NjMyM2ZkNjBlMWMxODVjZjYzNzg1ZmU4MWY1NmE2YzE');
 });
 
+// Add the code redirects for community-and-contribution pages
+
 app.get('/community/contribution-survey', (req, res) => {
```
Comment (Tosin-Balogun) [inline app.js:138]: Does this also need to change? @davidhunter08

File: app.js (hunk 1 above)
Comment (davidhunter08) [reply app.js:163]: LGTM

File: app.js (hunk 2 above)
Comment (davidhunter08) [reply app.js:138]: Doesn't look like it, as we don't link to it from anywhere in the manual. It should be ok leaving it how it is.

**Review by davidhunter08 — APPROVED** [review]
LGTM

**Pull request merged**